pkg/plugins/crypto/crypto.go

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
@Baha-sk I might need to pick on the term `plugins` now. There are "Go Plugins" and these are not. I think it could be confusing to re-use this term.

These packages are actually the default framework packages for the SPIs.

Reply:
I didn't think much about the `plugin` package name.

now that we have pkg/framework/spi for service plugin interfaces... we probably need a proper location for the implementations, including this default one. They should be under pkg/framework as these are Aries pluggable services.

Reply:
should be under pkg/framework/foo/package (rather than directly under framework), so that we match the spi structure.

Reply:
Not impl - we should make it clear that is only defaults.

Reply:
perhaps a parent folder name like `provider` to match service **provider** interface?

Reply:
so default implementations go in `pkg/framework/provider/` and interfaces stay in `pkg/framework/spi` ?

Reply:
Yes. 

Note: I assume packages in these folders are adapters. Major components like JWE should be in their own package elsewhere in the repo. These provider packages should import these major components.

Reply:
JWE package is one of these providers, just like crypto and KMS.

as depicted in the [use case](https://wiki.hyperledger.org/display/ARIES/Crypto+and+KMS+Plugins) diagram, JWE will use Crypto and KMS. 

In the end all three providers are services available to the framework, this is why they're all part of SPI/providers.. I don't see JWE as a major (separate) component that differs in nature from other providers/SPIs.